examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,269 @@
+# Backup and restore MySQL DB using CSI VolumeSnapshot
+
+VolumeSnapshots provide Kubernetes users with a standardized way to copy a volume's contents at a particular point in time without creating an entirely new volume. This functionality enables, for example, database administrators to backup databases before performing edit or delete modifications.
+
+## Introduction
+This document explains how Kanister leverages the use of CSI VolumeSnapshots to take backup and restore of databases in MySQL.
```

Comment:
```suggestion
This document explains how Kanister leverages the use of CSI VolumeSnapshots to snapshot and restore MySQL database.
```

Reply:
How about saying "....CSI VolumeSnapshots to **capture** and restore MySQL database"? Using snapshot again doesn't sound good when you read it.